sriov_network_operator/sriov_network_operator_ci_test.sh

```diff
@@ -16,22 +16,41 @@ function exit_code {
     exit $rc
 }
 
-function main {
-    status=0
+function test_sriov_operator_e2e {
+    pushd $WORKSPACE/sriov-network-operator
+
+    # Sometimes the config daemon fails to discover the card
+    # pfs, until the root cause is determined, deleting the
+    # the config daemon pod would solve the issue.
+
+    local config_daemon_info=$(kubectl get pods -A | grep sriov-network-config-daemon-cxstv)
```

Comment:
It won't work, pod name suffix is autogenerated each run. Let's get pods by labels:
kubectl get pods -A  -l app=sriov-network-config-daemon

Reply:
Thanks for the review, 
it was a mistake, will do.